xml/common_criteria.xml

```diff
@@ -102,9 +102,9 @@
 
    <para>
     While EAL1 only provides basic assurance for products to meet security
-    requirements, EAL2 to 4 are medium assurance levels. EAL5-EAL7 describe
+    requirements, EAL2 to EAL24 are medium assurance levels. EAL5 to EAL7 describe
```

Comment:
Rejecting and reverting this change; the max level is only 7, not 24. 

Citation: https://en.wikipedia.org/wiki/Evaluation_Assurance_Level